grid2op/data_test/l2rpn_idf_2023_with_alert/config.py

```diff
@@ -9,6 +9,8 @@
 try:
     from grid2op.l2rpn_utils import ActionIDF2023, ObservationIDF2023
 except ImportError:
+    from grid2op.Observation import CompleteObservation
+    import warnings
```

Comment:
Nice catch, can you check if it's correctly everywhere ? In the input_data_* of the starting kit and in the training environment of l2rpn_idf_2023 ?